pkgs/development/compilers/ghc/8.0.1.nix

```diff
@@ -33,7 +33,7 @@ stdenv.mkDerivation rec {
     (fetchFilteredPatch { url = https://git.haskell.org/ghc.git/patch/2f8cd14fe909a377b3e084a4f2ded83a0e6d44dd; sha256 = "06zvlgcf50ab58bw6yw3krn45dsmhg4cmlz4nqff8k4z1f1bj01v"; })
   ] ++ stdenv.lib.optional stdenv.isLinux ./ghc-no-madv-free.patch;
 
-  buildInputs = [ ghc perl hscolour sphinx];
+  buildInputs = [ ghc perl (stdenv.lib.getBin hscolour) sphinx];
```

Comment:
Hmm. Typically the way output propagation done by the stdenv multiple-outputs code works is that bringing the `dev` output to scope also brings `bin`.

E.g. `curl` has:
````
outputs = [ "bin" "dev" "out" "man" "devdoc" ];
````
but still adding `curl` to `buildInputs` adds the `curl` binary to PATH.

With that said, of course only adding `bin` might reduce the build-time dependency closure significantly.